delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/wiring/DeltaModule.scala

```diff
@@ -51,26 +51,37 @@ class DeltaModule(appCfg: AppConfig, config: Config)(implicit classLoader: Class
 
   make[List[PluginDescription]].from { (pluginsDef: List[PluginDef]) => pluginsDef.map(_.info) }
 
-  make[RemoteContextResolution].named("aggregate").fromEffect { (otherCtxResolutions: Set[RemoteContextResolution]) =>
-    for {
-      errorCtx      <- ContextValue.fromFile("contexts/error.json")
-      metadataCtx   <- ContextValue.fromFile("contexts/metadata.json")
-      searchCtx     <- ContextValue.fromFile("contexts/search.json")
-      tagsCtx       <- ContextValue.fromFile("contexts/tags.json")
-      versionCtx    <- ContextValue.fromFile("contexts/version.json")
-      offsetCtx     <- ContextValue.fromFile("contexts/offset.json") // TODO: Should be moved to views?
-      statisticsCtx <- ContextValue.fromFile("contexts/statistics.json") // TODO: Should be moved to views?
-    } yield RemoteContextResolution
-      .fixed(
-        contexts.error      -> errorCtx,
-        contexts.metadata   -> metadataCtx,
-        contexts.search     -> searchCtx,
-        contexts.tags       -> tagsCtx,
-        contexts.version    -> versionCtx,
-        contexts.offset     -> offsetCtx,
-        contexts.statistics -> statisticsCtx
-      )
-      .merge(otherCtxResolutions.toSeq: _*)
+  many[MetadataContextValue].addEffect(MetadataContextValue.fromFile("contexts/metadata.json"))
+
+  make[MetadataContextValue]
```

Comment:
this is not needed as you can just have the `Set[MetadataContextValue]` as a dependency below...

Reply:
I know is not needed, but it is useful since it might be used in the future by some other plugin. So instead of each plugin having to combine the `Set[MetadataContextValue]` they can use `metadataContext: MetadataContextValue @Id("aggregated-metadata")`